src/find.rs

```diff
@@ -101,10 +97,56 @@ impl PlayerFinder {
         }
     }
 
-    fn active_player_bus(&self) -> Result<Option<String>, FindingError> {
-        // Right now, we just pick the first of the players. Is there some way to select this more
-        // intelligently?
-        Ok(self.all_player_buses()?.into_iter().next())
+    /// Try to find the "active" [`Player`] in the connection.
+    ///
+    /// This method will try to determine which player a user is most likely to use. First it will look for a player with
+    /// the playback status [`Playing`](PlaybackStatus::Playing), then for a [`Paused`](PlaybackStatus::Paused), then one with
+    /// track metadata, after that it will just return the first it finds. [`NoPlayerFound`](FindingError::NoPlayerFound) is returned
+    /// only if there is no player on the DBus.
+    pub fn find_active<'b>(&self) -> Result<Player<'b>, FindingError> {
+        let mut players: Vec<Player> = self.find_all()?;
+
+        // Return Error if no players
+        if players.is_empty() {
+            return Err(FindingError::NoPlayerFound);
+        }
+
+        // Look for "Playing"
+        for (n, player) in players.iter().enumerate() {
+            if let PlaybackStatus::Playing = player.get_playback_status()? {
+                return Ok(players.remove(n));
+            }
+        }
+
+        // Look for "Paused"
+        for (n, player) in players.iter().enumerate() {
+            if let PlaybackStatus::Paused = player.get_playback_status()? {
+                return Ok(players.remove(n));
+            }
+        }
+
+        // Look for player with metadata
+        for (n, player) in players.iter().enumerate() {
+            if !player.get_metadata()?.as_hashmap().is_empty() {
+                return Ok(players.remove(n));
+            }
+        }
```

Comment:
This could also check for a valid `TrackID` in the `Metadata` but I don't think it really matters. Might be a bit more efficient but it would fail if a player doesn't set that value properly.